services/movie.py

```diff
@@ -15,13 +17,17 @@
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__contains=title)
+
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
```

Comment:
Using `get` without handling exceptions can lead to `DoesNotExist` errors if the movie ID is not found. Consider using `get_object_or_404` if this is used in a view, or handle the exception to provide a more graceful error handling.